collect_app/src/main/java/org/odk/collect/android/activities/GeoPointMapActivity.java

```diff
@@ -496,7 +466,7 @@ public void onCancel(DialogInterface dialog) {
             if (location != null) {
                 zoomLocationButton.setEnabled(true);
                 zoomLocationButton.setBackgroundColor(Color.parseColor("#50cccccc"));
-                zoomLocationButton.setTextColor(themeUtils.getPrimaryTextColor());
+                zoomLocationButton.setTextColor(Color.parseColor("#ff333333"));
```

Comment:
These changes aren't lambda changes so probably shouldn't be in this commit (and likely not this PR?).

Reply:
Thanks for spotting these.These changes aren't supposed to here so something got messed up when I was cloning master so making the fixes again.